app/models/requirement_block.rb

```diff
@@ -27,11 +27,15 @@ def search_data
     }
   end
 
+  def key
+    "formSubmissionDataRequirementblock#{id}"
```

Comment:
Will this cause an issue if a form template has different sections with same requirement block? Maybe have to move this to the join table function or pass in the section id in the to form json function from the sections

Reply:
good pint, will add the section as well here.